### ISSUE TITLE
[FEATURE] Add priority numbers to catalog schemas.

### ISSUE THREAD

**Issue by Swiddis** [opensearch-project/opensearch-catalog#37]
### Is your feature request related to a problem?
When a user attempts to manually create an integration index template after automatic creation via the available options, there is a conflict on the matching index patterns. This manifests as an error resembling:

```
[illegal_argument_exception]: index template [logs] has index patterns [ss4o_logs-*-*] matching patterns from existing templates [logs_1.0.0] with patterns (logs_1.0.0 => [ss4o_logs-nginx-*-sample]) that have the same priority [0], multiple index templates may not match during index creation, please use a different priority
```

### What solution would you like?
While code can be implemented to apply priority numbers to sample indices, the most stable solution is to have them available as part of the schemas already. This was the route taken with the local copies of the schemas used in the integrations project. This is as easy as adding a `"priority": 900` line to every schema in the repo.

### What alternatives have you considered?
It could be handled in-code. As mentioned, it's more stable and configurable to have it assigned in the repo as-is.

### Do you have any additional context?
N/A

**Comment by YANG-DB** [comment]
I dont think this is the solution here - if an existing log patterns is conflicting with a new pattern - user has to resolve the pattern conflict rather than assume some priority would save all the different cases in a similar manner.
a user should reconcile the patterns according to his own needs - this can be done using the index management plugin configuration
